nutanix/resource_nutanix_foundation_ipmi_config.go

```diff
@@ -170,6 +169,24 @@ func resourceFoundationIPMIConfigCreate(ctx context.Context, d *schema.ResourceD
 		return diag.FromErr(err)
 	}
 
+	// incase of errored response
+	if resp.Error != nil {
+		// check if error details exists for every ipmi IP
+		var diags diag.Diagnostics
+		for k, v := range resp.Error.Details {
+			diags = append(diags, diag.Diagnostic{
```

Comment:
same as above

Reply:
diag.Errorf() returns diag.Diagnostics, but we need to create a diag.Diagnostic for every node failure and eppend them to create a diag.Diagnostics